app/src/release/java/com/futurice/freesound/app/module/InstrumentationModule.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2016 Futurice GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.futurice.freesound.app.module;
+
+import java.util.Collections;
+import java.util.List;
+
+import javax.inject.Singleton;
+
+import dagger.Module;
+import dagger.Provides;
+import okhttp3.Interceptor;
+
+@Module
+public final class InstrumentationModule {
```

Comment:
I think testing this is a great idea so will add tests. 

However, I am a bit uncertain about the AppInterceptors provider right now. I assumed it's just a placeholder for now and it's supposed to return something in the future, since right now both `release` and `debug builds both return empty lists. Of course I can add a test for this too in `testRelease` but should it be empty in the future as well?